app/src/test/java/waiter/ClientConnectionTest.java

```diff
@@ -14,15 +16,23 @@ class ClientConnectionTest {
     @Test
     void readsMessage() throws IOException {
         String input = "foo";
-        String message = new ClientConnection(new InputStreamerMock(input), new OutputStreamerMock()).read().open();
+        String message = new ClientConnection(new InteractorMock(), new InputStreamerMock(input), new OutputStreamerMock()).read();
         assertEquals(message, input);
     }
 
     @Test
     void writesMessage() throws IOException {
         OutputStreamerMock outputStreamerMock = new OutputStreamerMock();
-        ClientConnection clientConnection = new ClientConnection(new InputStreamerMock("foo"), outputStreamerMock);
-        clientConnection.write(new Message("foo"));
+        ClientConnection clientConnection = new ClientConnection(new InteractorMock(), new InputStreamerMock("foo"), outputStreamerMock);
```

Comment:
Same here

Reply:
Addressed in [this commit](https://github.com/jonmanahan/waiter/pull/7/commits/713d471c9fe377388299139e90627d4683daca6c), formatted to have the input -> exercise -> assert clearly separated 👍